lib/rules/web/admin_console/4.11/dashboards.xyaml

```diff
@@ -146,7 +142,6 @@ change_workload:
   action: click_button_text
 check_dashboard_dropdown_items:
   action: click_dashboard_dropdown
-  action: check_dashboard_etcd
```

Comment:
on hypershift provisioned clusters, no `etcd` dashboard
so remove `etcd` dashboard check on 4.11 and 4.12